testing/spectest/shared/common/forkchoice/builder.go

```diff
@@ -144,9 +157,9 @@ func (bb *Builder) Check(t testing.TB, c *Check) {
 	}
 	if c.ProposerBoostRoot != nil {
 		want := fmt.Sprintf("%#x", common.FromHex(*c.ProposerBoostRoot))
-		bb.service.ForkChoiceStore().RLock()
```

Comment:
Why is there a `ForkChoiceStore` and `ForkChoicer`?

Reply:
You can remove one, they are exactly the same method defined in the same file. I do not know from where they are. We should in fact try to remove them both and only export locked wrappers to forkchoice